components/apimgt/org.wso2.carbon.apimgt.persistence.mongodb/src/main/java/org/wso2/carbon/apimgt/persistence/mongodb/MongoDBAtlasAPIConnector.java

```diff
@@ -0,0 +1,180 @@
+/*
+ *  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
license header year to 2021? If you created this during last month (dec 2020) you can keep as it is.